GTG/core/meson.build

```diff
@@ -1,3 +1,16 @@
+infodir = python3.get_install_dir()
+version = run_command('git', 'describe', '--dirty').stdout().strip()
+
+info_config = configuration_data()
+info_config.set('VCS_TAG', version)
+
+info_py = configure_file(
+            input : 'info.py.in',
+            output :'info.py',
+            configuration: info_config,
+            install : true,
+            install_dir: infodir / 'GTG/core')
```

Comment:
I think you should also use the `/` operator here, just in case: `infodir / 'GTG' / 'core'`